controllers/genres.js

```diff
@@ -0,0 +1,43 @@
+const models = require('../models')
+
+const getAllNovelsAndAuthorsInAllGenres = async (request, response) => {
+  try {
+    const novelsInGenres = await models.Genres.findAll()
+
+    return (novelsInGenres)
+      ? response.send(novelsInGenres)
+      : response.sendStatus(404)
+  } catch (error) {
+    response.status(500).send('Unable to retrieve genres, please try again')
+  }
+}
+
+const getNovelsAndAuthorsByGenre = async (request, response) => {
+  try {
+    const { id } = request.params
+
+    const novelsAndAuthorsByGenre = await models.Genres.findOne({
+      where: { Id: id },
+      include: [
+        {
+          model: models.Novels,
+          include: [
+            { model: models.Authors },
+            {
+              model: models.Genres,
+              where: { id },
+              attributes: []
+            }]
```

Comment:
The response might be misleading, I wouldn't filter out genre here, you likely want all the other genres for the novel. You technically don't need the one requested because if the novel wasn't of that genre you wouldn't see the novel since you've requested the novels of the genre. So including genre again only to restrict to the one feels odd.